settings/arm9/source/main.cpp

```diff
@@ -198,12 +198,13 @@ void loadROMselect()
 	}
 }
 
-void loadThemeList(const char* directory, std::vector<std::string>& list)
```

Comment:
@NightYoshi370 If you change this to ` void loadThemeList(const char* directory, std::vector<std::string>* list)`